packages/liveblocks-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@liveblocks/node",
-  "version": "0.3.0",
+  "version": "0.17.0-test1",
   "description": "A server-side utility that lets you set up a Liveblocks authentication endpoint.",
   "main": "./lib/cjs/index.js",
   "module": "./lib/esm/index.js",
```

Comment:
Oh, btw. If we're going to do this, it may be nice to invest a bit more in the build process and use the same build process we use for the other packages, so it's all unified and there's no different setup for this package anymore. Not sure if there is a reason this package uses different build steps. Perhaps it's out of necessity but it might as well be legacy? If the latter, let's keep it on par with the rest while we're at it.

To clarify, what I mean is using Rollup to produce the build (instead of TypeScript), and output these in the same "flattened" package output (no different ESM/CJS directories)—just like all the other packages.

I think the `rollup.config.js` files from the other packages are generic enough to be copy/pastable in here.

Reply:
Thanks for the tips! I've adjusted the PR to reflect that, `npm run build` works succesfully with `rollup` now. 
I re-used the config from `liveblocks-client` to achieve this and slightly modified it to fit the needs for `liveblocks-node`.